crates/hyperqueue/src/common/cli.rs

```diff
@@ -185,35 +185,35 @@ pub struct RootOptions {
 #[allow(clippy::large_enum_variant)]
 #[derive(Parser)]
 pub enum SubCommand {
-    /// Commands for controlling the HyperQueue server
+    /// Commands for the server
     Server(ServerOpts),
-    /// Commands for controlling HyperQueue jobs
+    /// Commands for jobs
     Job(JobOpts),
-    /// Commands for displaying task(s)
+    /// Commands for tasks
     Task(TaskOpts),
-    /// Submit a job to HyperQueue
+    /// Submit a new job
     Submit(JobSubmitOpts),
-    /// Commands for controlling HyperQueue workers
+    /// Commands for workers
     Worker(WorkerOpts),
-    /// Operations with log
+    /// Operations for streaming logs
     OutputLog(OutputLogOpts),
     /// Automatic allocation management
     #[command(name = "alloc")]
     AutoAlloc(AutoAllocOpts),
-    /// Event and journal management
+    /// Journal management
     Journal(JournalOpts),
     /// Data object management inside a task
     Data(DataOpts),
-    /// Start the HyperQueue CLI dashboard
+    /// Starts CLI dashboard
```

Comment:
On a bunch of places, there's inconsistency between "this command does something" vs "do something".

e.g. "starts dashboard" vs "show documentation"

But it's not super important.